index.html

```diff
@@ -50,100 +40,94 @@ <h1 class="title">Isaias Del Mundo</h1>
             <img src="Ryu.JPG" alt="ryus_world">
         </div>
         <div class="column">
-            <span class="title" id="aboutMe">About Me</span><br>
-            <span class="subtitle">I, Isaias Del Mundo, am a full stack developer living in Hayward, California. I have a certiciate in Electronic technology and Automotive technology from College of San Mateo and Wyotech. Currently enrolled at the UC Berkeley extension Full Stack developer. I chose this school becauce its associated with UC Berkeley and i think thats prety cool having that in my resume. </span>            <br>
-            <hr>
-            <span class="subtitle"> I've been in automotive industry since I was 16 years old and moving into a progorammer world is pretty interesting. I love cars and it gets annoying when things don't go the right way just like coding. Nothing brings me more pleasure than making something from scratch. Even when the result are far from what my ideal expectaions.</span>            <br>
-            <hr>
-            <span class="subtitle"> Life has been full of surpises I've never thought of being laid off from work but things happens for a reason. This pandamic has been tough but I managed to make it better by learning something completely different. I hope to find a job once I am done with school.</span>
-
+            <span class="title" id="aboutMe">About Me</span>
+            <span class="subtitle">I, Isaias Del Mundo, am a full stack developer living in Hayward, California. I have a certiciate in Electronic technology and Automotive technology from College of San Mateo and Wyotech. Currently enrolled at the UC Berkeley extension Full Stack developer. I chose this school becauce its associated with UC Berkeley and i think thats prety cool having that in my resume.</span>
```

Comment:
```suggestion
            <p class="subtitle">I, Isaias Del Mundo, am a full stack developer living in Hayward, California. I have a certificate in Electronic technology and Automotive technology from College of San Mateo and Wyotech. Currently enrolled at the UC Berkeley extension Full Stack developer. I chose this school because its associated with UC Berkeley and i think thats pretty cool having that in my resume.</p>
```
Use a `p` tag instead of a `span`